docs/ref/models/conditional-expressions.txt

```diff
@@ -141,6 +141,17 @@ the ``Client`` has been with us, we could do so using lookups::
     both Jane Doe and Jack Black. This works just like an :keyword:`if` ...
     :keyword:`elif` ... :keyword:`else` statement in ``Python``.
 
+``Case()`` also works with ``filter()`` to fetch objects::
+
+    >>> a_month_ago = date.today() - timedelta(days=30)
```

Comment:
You didn't use `a_month_ago`. Is it a realistic query if both conditions use `a_year_ago`?